benchmarks/BoltzmannWealth/boltzmann_wealth.py

```diff
@@ -74,9 +75,11 @@ def run_model(self, n):
             self.step()
 
 
-class MoneyAgent(mesa.Agent):
+class MoneyAgent(mesa.Agent, HasObservables):
```

Comment:
How expensive would it be for `mesa.Agent` to automatically inherit `HasObservables`? The less boilerplate, the better.

Reply:
It would not be expensive at all. The same applies to Model, and possible in the future the spaces. However, I have atm designed everything to be in experimental so as not to interfere with the stable code.